core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -724,13 +709,44 @@ package object config {
       .checkValue(v => v > 0, "The max failures should be a positive value.")
       .createWithDefault(40)
 
-  private[spark] val EXECUTOR_PLUGINS =
-    ConfigBuilder("spark.executor.plugins")
-      .doc("Comma-separated list of class names for \"plugins\" implementing " +
-        "org.apache.spark.ExecutorPlugin.  Plugins have the same privileges as any task " +
-        "in a Spark executor.  They can also interfere with task execution and fail in " +
-        "unexpected ways.  So be sure to only use this for trusted plugins.")
-      .stringConf
-      .toSequence
-      .createWithDefault(Nil)
+  private[spark] val EXECUTOR_PLUGINS = ConfigBuilder("spark.executor.plugins")
+    .doc("Comma-separated list of class names for \"plugins\" implementing " +
+      "org.apache.spark.ExecutorPlugin.  Plugins have the same privileges as any task " +
+      "in a Spark executor.  They can also interfere with task execution and fail in " +
+      "unexpected ways.  So be sure to only use this for trusted plugins.")
+    .stringConf
+    .toSequence
+    .createWithDefault(Nil)
+
+  private[spark] val MASTER_REST_SERVER_ENABLED = ConfigBuilder("spark.master.rest.enabled")
+    .booleanConf
+    .createWithDefault(false)
+
+  private[spark] val MASTER_REST_SERVER_PORT = ConfigBuilder("spark.master.rest.port")
+    .intConf
+    .createWithDefault(6066)
+
+  private[spark] val MASTER_UI_PORT = ConfigBuilder("spark.master.ui.port")
+    .intConf
+    .createWithDefault(8080)
+
+  private[spark] val ADMIN_ACLS = ConfigBuilder("spark.admin.acls")
```

Comment:
In spite of the name, all of these ACL-related options are UI options.

Reply:
I thought these options are common thing between UI and History, but looks like History has its own ACL configurations. Will move to UI.